server/src/database/models/course.ts

```diff
@@ -0,0 +1,43 @@
+// SPDX-FileCopyrightText: 2022 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+import { DataTypes, Model, InferAttributes, InferCreationAttributes, CreationOptional } from 'sequelize';
+import { sequelize } from '..';
+
+export default class Course extends Model<InferAttributes<Course>, InferCreationAttributes<Course>> {
+  declare id: CreationOptional<number>;
```

Comment:
Does this mean that the PK is nullable?

Reply:
Non nullable, on Sequelize level PK is not required since postgres takes care of assigning id (sequence)